assets/css/custom.css

```
@@ -318,6 +318,40 @@ html { scroll-behavior: smooth; }
+.repo-topics { display: flex; flex-wrap: wrap; gap: .4rem; margin-top: .8rem; }
+.repo-topic { font-size: .78rem; color: rgb(var(--color-primary-300)); text-decoration: none; }
+.repo-topic:hover { text-decoration: underline; }
+.repo-body { margin-top: 1rem; }
+
+.repo-grid { display: grid; gap: 1rem; grid-template-columns: repeat(auto-fill, minmax(260px, 1fr)); margin: 1.5rem 0; }
+.repo-card { display: flex; flex-direction: column; gap: .4rem; padding: 1rem; border-radius: var(--radius);
+  background: rgb(var(--bg-elevated) / .6); border: 1px solid rgb(var(--color-neutral-700) / .5);
+  text-decoration: none; color: inherit; transition: transform .15s ease, border-color .15s ease, background .15s ease; }
+.repo-card:hover { transform: translateY(-3px); border-color: rgb(var(--ring) / .6); background: rgb(var(--bg-elevated) / .9); }
+.repo-card-top { display: flex; align-items: baseline; justify-content: space-between; gap: .5rem; }
+.repo-card-name { font-weight: 700; color: rgb(var(--color-primary-200)); font-size: .98rem; word-break: break-word; }
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)